index.js

```diff
@@ -550,19 +535,8 @@ Client.prototype.isValidPassPhrase = function (passPhrase) {
     return false
   }
 
-  const passPhraseWords = passPhrase.split(' ')
-
-  if (![NICEWARE_PASSPHRASE_LENGTH, BIP39_PASSPHRASE_LENGTH].includes(passPhraseWords.length)) {
-    this.memo('isValidPassPhrase', `invalid passphrase: must be either ${BIP39_PASSPHRASE_LENGTH} or ${NICEWARE_PASSPHRASE_LENGTH} words`)
-    return false
-  }
-
   try {
-    if (passPhraseWords.length === BIP39_PASSPHRASE_LENGTH) {
-      Buffer.from(bip39.mnemonicToEntropy(passPhrase), 'hex')
-    } else {
-      niceware.passphraseToBytes(passPhraseWords)
-    }
```

Comment:
^ code above not needed because of https://github.com/brave/crypto/blob/master/index.js#L205-L210